### PR TITLE
Fix CreateCity form not having data

### PR DESCRIPTION
**Pull request by tvardero**
[#2484](https://github.com/ita-social-projects/EPlast/issues/2484)

### CODE REVIEW

**Review by Goven112 — APPROVED** [review]

**Review by bkondrat — APPROVED** [review]

File: src/pages/City/CreateCity/CreateCity.tsx [hunk 1]
```diff
@@ -68,7 +68,9 @@ const CreateCity = () => {
   const history = useHistory();
   const location = useLocation();
   const followerPath = "/regions/follower/";
-  const [loading, setLoading] = useState(false);
+  const isFollowerPath = location.pathname.includes(followerPath);
+
+  const [loaded, setLoaded] = useState<boolean>(false);
```
Comment (oshut) [inline src/pages/City/CreateCity/CreateCity.tsx:73]: If state are `boolean` it's a good practice to name it with `is` prefix
For example - `isLoaded`

File: src/pages/City/CreateCity/CreateCity.tsx [hunk 2]
```diff
@@ -80,26 +82,29 @@ const CreateCity = () => {
...
-      if (location.pathname.startsWith(followerPath + "edit")) {
-        getRegionFollower(id);
-      } else {
-        getActiveUser();
-      }
-      getRegions();
-    } else {
-      if (+id) {
-        getCity().then(() => getRegions());
+  const load = async () => {
```
Comment (oshut) [inline src/pages/City/CreateCity/CreateCity.tsx:86]: `load` what?

Comment (tvardero) [reply src/pages/City/CreateCity/CreateCity.tsx:86]: All needed data from backend
I'll improve naming.

File: src/pages/City/CreateCity/CreateCity.tsx (hunk 1 above)
Comment (tvardero) [reply src/pages/City/CreateCity/CreateCity.tsx:73]: Ok

**Review by Omeliukh-Nazar — APPROVED** [review]

**Pull request merged**